bolides.js

```diff
@@ -467,13 +549,14 @@ var bolides = {
         } else if (bolides.asteroid3.y <= -50) {
             bolides.asteroid3.isInMotion = false;
         }
+        /*
         // Collision detection
```

Comment:
If it helps, here's a super basic rectangular collision detection method: 

https://github.com/greypants/blastEngine/blob/master/javascript/engine/game.js#L37

I recommend abstracting your collision logic into a reusable function that takes two objects, compares them to see if there's a collision, and returns `true` or `false`